lib/pbkit/nv_objects.h

```diff
@@ -1219,8 +1219,11 @@ NV20_TCL_PRIMITIVE_3D      [0x1d78/4] = 0x00000001 | UNKNOWN = 00000001
 #define NV20_TCL_PRIMITIVE_3D_SHADER_TYPE				0x00001e94
 #define 	SHADER_TYPE_INTERNAL	4
 #define 	SHADER_TYPE_EXTERNAL	6
+#define 	SHADER_TYPE_INTERNAL_LIMITED	0
+#define 	SHADER_TYPE_EXTERNAL_LIMITED	2
 #define NV20_TCL_PRIMITIVE_3D_SHADER_SUBTYPE				0x00001e98
 #define		SHADER_SUBTYPE_REGULAR	0 						//not allowed to write into constants: faster
+#define		SHADER_SUBTYPE_WRITE		1 	  		  	//can write into constants: slower?!
```

Comment:
change tabs to spaces?
(existing code used tabs, but it looks like a giant mess)